app/helpers/object_link_helper.rb

```diff
@@ -7,7 +7,7 @@ module ObjectLinkHelper
     Google_Maps: "https://maps.google.com/maps?q=",
     Google_Search: "https://www.google.com/search?q=",
     Wikipedia: "https://en.wikipedia.org/w/index.php?search="
-  }.freeze
+  }
```

Comment:
Redundant (and offend RuboCop) in light of magic comment at top of file: `# frozen_string_literal: true`